packages/api-request-builder/test/query-id.spec.js

```diff
@@ -34,12 +34,29 @@ describe('queryId', () => {
     expect(service.params.customerId).toBe('myCustomer')
   })
 
+  test('should set the orderNumber param', () => {
```

Comment:
nitpick: It would be nice to move this block down below the test for `customerId` so they are all organized together.

Reply:
I've moved it, take another look.